src/components/ChecklistPanel.js

```diff
@@ -86,6 +87,33 @@ class ChecklistPanel extends Component {
 		if ( prevProps.isPublishSidebarEnabled !== current && ! current ) {
 			this.props.onEnablePublishSidebar();
 		}
+		// Toggle the post saving lock.
+		this.toggleSavingLock();
+	}
+
+	toggleSavingLock() {
```

Comment:
`toggle` to me implies it's going to change it, whereas I think this is more like `maybeUpdateSavingLock` or `checkSavingLock`?